Fix venv instructions (#6675)
scikit-image/scikit-image: CONTRIBUTING.rst

       ::

         # Create a virtualenv named ``skimage-dev`` that lives outside of the repository.

>          # One convention is to use an ``envs`` directory under the home directory:

Reviewer: ```suggestion
         # One common convention is to place it inside an ``envs`` directory under your home directory:
```

>          # Create a virtualenv named ``skimage-dev`` that lives in a dedicated directory outside
>          # the source tree. One suggestion below is to create an ~/envs folder:

Reviewer 2: ```suggestion
         # Create a virtualenv named ``skimage-dev`` that lives outside of the repository.
         # One convention is to use an ``envs`` directory under the home directory:
```

>          mkdir -p ~/envs

Reviewer: ```suggestion
         mkdir ~/envs
```
